scripts/run-pre-build-check.sh

```diff
@@ -15,9 +15,15 @@ else
   PLATFORM=$1
 fi
 
-npm_version=$(npm -v)
-if [[ $npm_version != "5.5.1" ]]; then
-  echo -e "${YELLOW}+ npm version $npm_version is installed. npm version 5.5.1 is recommended.${NC}"
+node_version=$(node -v)
+
+if [[ $node_version != "v9.3.0" ]]; then
```

Comment:
we could extract the expected version numbers as variables at the head of the script

Reply:
fixed it

Reply:
sorry for force-push, but I needed to prepare it for merging :)